src/main/java/io/turntable/projectwebservice/controllers/ProjectController.java

```diff
@@ -58,21 +62,22 @@ public void updateProjectRecord(@PathVariable String id, @RequestBody Project pr
         projectToUpdate.setProject_name(project.getProject_name());
         projectToUpdate.setDescription(project.getDescription());
         projectService.updateProject(projectToUpdate);
+        Publisher.sendMessage(String.format("project with id = %s, name = '%s' updated [UPDATE]", projectToUpdate.getProject_id(), project.getProject_name()));
     }
 
 
     @ApiOperation("get project by id")
     @GetMapping("/project/search/id/{id}")
     @ResponseStatus(HttpStatus.OK)
     public Project getProjectById(@PathVariable String id) {
```

Comment:
you could have mage the id input as "Integer" as a way of safe checking and spring does check & converting for you automatically!. Since its string, I can pass any rubbish as a string for id

Reply:
better still a "Long" data type

Reply:
oh....yeah.....i see other aliens use Long for id most open.....what's that thing with id with "long"....can it be that Long ... hehe?